netci.groovy

```diff
@@ -71,7 +71,7 @@ static void addUnitPublisher(def myJob) {
       'xunit'('plugin': 'xunit@1.97') {
       'types' {
         'XUnitDotNetTestType' {
-          'pattern'('**/*TestResults.xml')
+          'pattern'('**/xUnitResults/*.xml')
```

Comment:
Because our internal tests already separate logs out into a separate folder. This also makes them easier to locate on your box :)

Also the job regeneration is cheap (it takes less than a minute and updates all the jobs simultaneously). It shouldn't matter if we update it multiple times.